source/store/AuthContext.js

```diff
@@ -162,28 +180,58 @@ function AuthProvider({ children, initialState }) {
       return new Date().getTime() < expiresAt;
     }
     return false;
-  }
+  }, [handleSetMode]);
+
+  useEffect(() => {
+    const tryFetchUser = async () => {
```

Comment:
I tink the nameing is somewhat unclear. The code block is telling me we're handling a profile of some kind?
Maybe the naming could be named to something closer to what's happeing between the if block?
Have no suggestion by my own. Sorry. Just a matter of taste :)

Reply:
Will rename tryFetchUser to trySignIn